src/client/pages/ConsentsOurContent.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
```

Comment:
Can we get a storybook for this page please!

Reply:
Yes, this has actually turned out to be a bit more complicated than I first thought. It seems adding the check for an AB test here: `const ABTestAPI = useAB();` breaks the story book components. 

I've moved the check to further up the pages here: https://github.com/guardian/gateway/pull/2349 but this does introduce a code change to a lot of different files, so I'm not sure if this is the best approach

Reply:
That's actually the approach I'd have suggested, add the `useAB` check on the `*Page.tsx` file, and the component itself takes a prop to determine if it's in the test or not.

The components should be able to exist without any external api/state etc being needed to be call, anything they need should be passed in as props.

Reply:
Ok yeah, fair enough. We can take this approach in that case

Reply:
If we use the `clientState` to figure out the AB test instead then this is what the changes look like: https://github.com/guardian/gateway/pull/2350

It's more convenient from a code point of view, but I noticed it says it's not a recommended approach from here: https://github.com/guardian/gateway/blob/d5ceda4ecbe40aba7b8816d9ef5775b296dc6256/src/client/components/ABTestDemo.tsx#L56C1-L64C26

Would there be any concerns with taking this approach?